config/routes.rb

```diff
@@ -0,0 +1,23 @@
+Rails.application.routes.draw do
+  # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
+
+  root 'homepages#index'
+
+  resources :votes, :works
```

Comment:
`resources` produces every RESTful route for this resource. Do we need every RESTful route for votes?